stack.yaml

```diff
@@ -47,6 +52,7 @@ extra-deps:
     commit: e8bfc1294e088f90e5ae0b4aedbc82ee46ac5ee4
 
   - bimap-0.4.0
+  - binary-0.8.7.0
```

Comment:
Why is this needed?

Reply:
Whenever I don't have it, I can't build. Not sure why nobody else seems to have this problem...

Reply:
Aarrgh, this line has cost me soooo much time rebuilding when switching branches: stack always has to build +100 dependencies whenever the version of `binary` changes in stack.yaml. Why can't stack remember that it has built these dependencies before? (I'm not blaming you, just venting :slightly_smiling_face: )

Reply:
Use nix :)

Reply:
I would like to, but using `cabal new-build` in a nix-shell is still recompiling my dependencies, which is defeating the point :man_facepalming: